### DISCUSSION TITLE
auto_start_pw doesn't work.

### DISCUSSION THREAD

**Discussion by wab2** [kangyu-california/PersistentWindows#245] [Q&A]
When I try to double click or run as administrator, a cmd window pops up and says: 
Processing -File 'C:\Users\wab\Desktop\Persistent' failed because the file does not have a '.ps1' extension. Specify a valid Windows PowerShell script file name, and then try again.

[![image](https://github.com/kangyu-california/PersistentWindows/assets/141990159/7e4143cc-03ad-4e0c-85b0-05f78629e04e)
](https://prnt.sc/hFSicGcX10MB)

what should I do? I don't know how to make it auto-start manually via the task scheduler either.

**Comment by kangyu-california** [comment]
Thanks for the issue report,
the fix is to change the first line of auto_start_pw.bat 
from
powershell -ExecutionPolicy Bypass -File %~dp0\auto_start_pw_aux.ps1
to
powershell -ExecutionPolicy Bypass -File "%~dp0auto_start_pw_aux.ps1"

**Reply by wab2** [reply]
Could you please provide me with the exact steps to fix this? I am a beginner in this area

**Reply by kangyu-california** [reply]
Here is the latest patch [PersistentWindows5.43_0812.zip](https://github.com/kangyu-california/PersistentWindows/files/12328561/PersistentWindows5.43_0812.zip)

**Reply by EmilioMJ** [reply]
Patched worked. Thank you!
